packages/core/src/lib/notify.svelte

```diff
@@ -0,0 +1,127 @@
+<svelte:options immutable />
+
+<script lang="ts">
+type Variants = 'danger' | 'warning' | 'success' | 'info';
```

Comment:
Can you inline this type on L12? It will make IDE suggestions easier to use

Reply:
what do you mean by this?

Reply:
See below!